README.md

```diff
@@ -1,9 +1,14 @@
 # Link Extractor
 
-This repository is a [Docker](https://www.docker.com/) tutorial that illustrates some basics of Docker application containerization platform.
+Link Extractor app scrapes links from a given web page. This repository illistrates step by step approach to learn [Docker](https://www.docker.com/).
```

Comment:
Split sentences in separate lines (not mandatory, but is the undocumented chosen style in this repo). Also, fixing a typo.

```md
Link Extractor app scrapes links from a given web page.
This repository illustrates step by step approach to learn [Docker](https://www.docker.com/).
```